### PR TITLE
Pull Request to Review README.md Changes

### PR DESCRIPTION
**Pull request by mai-repo**
Hi, 

This Pull Request is looking at changes I made to my README.md for my Mai Blog application. Some of the changes includes adding in an image and removing a copy & paste link from a different repository. Let me know if you have any suggestions, or if they edits look good. Thanks!

### CODE REVIEW

**Comment by mai-repo** [comment]
This pull request is for @mx-ruthie.

**Comment by mai-repo** [comment]
@tati-guzman adding to this pull request

File: mai_blog/README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Planning stage for Mai BLog App
-![Image of my planning stage](/Planning_Stage.png)
+![Image of my planning stage](Planning_Stage.png)
 
 # Mai Blog 
 - A blog that documents all the places that I (Mai) have travelled to for users to see and review. You can also create a blog entry, though for personal purposes I have opted out of allowing users to delete any entries as these are personal memories, and like, memories they should not be deleted. They can be added to, similar to adding a memory.
```
Comment (tati-guzman) [inline mai_blog/README.md:5]: I would recommend moving the final comma in your second sentence to make it flow better:

"these are personal memories, and like memories, they should not be deleted"

**Review by tati-guzman — APPROVED** [review]
### Feedback

I would recommend doing a local branch merge as a separate PR in the future to be able to focus on the current README. For the Blog App README, it looks great so far! May be missing some details in sections you have outlined, but the plan is comprehensive and is setting you up well. I left a comment to move one comma, but that should be left to your discretion so I am submitting approval without need to address before merging. Great job!

**Review by mx-ruthie — COMMENTED** [review]
I strongly suggest y'all move the individual projects to their own repos. This diff is wild! 
Screenshot 2024-10-03 at 10 10 22 AM

When submitting commits, it's great to batch the commit with changes relenvant to each other. For example, a good commit message may be, "adds all api endpoints in the backend" or "connects all location buttons with their corresponding onclick functions" - so that each commit message says actively what it is changing about your code from the last working iteration. Then when something specific breaks, you know where to look in your commit histories.

**Comment by mai-repo** [comment]
@jsnorek can you please review?

**Pull request merged**